### DISCUSSION TITLE
Benchmark for Janusgraph

### DISCUSSION THREAD

**Discussion by trantuananh1** [JanusGraph/janusgraph#2648] [Q&A]
Which tool can I test performance of JanusGraph

**Comment by li-boxuan** [comment]
To my best knowledge, there is no official benchmark. You might want to check out some third-party benchmarks, for example, https://github.com/rngcntr/janusbench

**Reply by rngcntr** [reply]
I do not actively maintain this project anymore but if you would like to use it and have any questions, feel free to contact me anyway.
